src/openscad.cc

```diff
@@ -978,6 +978,7 @@ int main(int argc, char **argv)
     ("camera", po::value<string>(), "camera parameters when exporting png: =translate_x,y,z,rot_x,y,z,dist or =eye_x,y,z,center_x,y,z")
     ("autocenter", "adjust camera to look at object's center")
     ("viewall", "adjust camera to fit object")
+    ("disable-manifold", "render everything with CGAL (disables Manifold) for legacy, slower behaviour")
```

Comment:
Not sure we're ready to switch the default yet. It would be nice with an option where we could select backend, then switch the default value for that option down the road.

Reply:
Got it, switched it back to cgal + turned the UI to a combo (see screenshot in PR desc)